src/components/Kv/KvPillToggle.vue

```diff
@@ -1,34 +1,62 @@
 <template>
 	<div class="pill-toggle">
-		<span
-			v-for="option in options" :key="option.key"
-			@click="pillToggled(option.key)"
-			:class="[option.key, {active: selected === option.key }]"
+		<div
+			v-for="option in options"
+			:key="option.key"
+			class="pill"
 		>
-			<div class="title">
+			<input
+				class="radio"
+				type="radio"
+				:name="`pill-${uuid}`"
```

Comment:
why the uuid structure instead of just pulling index off of the v-for?

Reply:
It showed up when I put multiple kv-pill-toggle elements on a page that they'll have the same names / ids. So you click an input on the bottom kv-pill-toggle and the top one updates!

Reply:
Ahhh ok... So this is really a form input/label correlation specific issue. Just want to be careful of propagating this type of extra code throughout other components.

Should be ok with regard to form elements as there are only so many of them, though if it starts to go beyond that let's find a better way to eliminate the boilerplate.

Reply:
I don't like this. Is this a bug with storyBook? multiple components on the same page won't do this with Vue. If it is storybook related the fix should be in the story, not in the component

Reply:
@eddieferrer You're right! The bug wasn't in Storybook bug though, it was that my demo had all of the instances bound to the same Vue model. Doh! Will push a fix soon.

Reply:
Well now it seems like it was a combination of both duplicate model bindings and also duplicate IDs. By removing the IDs and for attributes on the labels and wrapping them instead it seems we're OK. For reference, here's a [paired down example](https://codesandbox.io/s/relaxed-cherry-1qfo2) of 
`<input id="test"><label for="test">`
 vs 
`<label><input></label>`.